### ISSUE TITLE
[Codelab]: TensorFlowJS Retrain a comment spam detection model to handle edge cases -- file not found

### ISSUE THREAD

**Issue by ConkyTheGreat** [tensorflow/tfjs#7388]
---
name: [Codelab]: TensorFlowJS Retrain a comment spam detection model to handle edge cases -- file not found
about: File not found when following codelab instructions
labels: 'type:bug'

---

**System information**
N/A

**Describe the current behavior**

When working through [this codelab example](https://codelabs.developers.google.com/tensorflow-retraining-comment-spam-detection#3), it instructs the user to use the following code:

`data_file = tf.keras.utils.get_file(fname='comment-spam-extras.csv', origin='https://storage.googleapis.com/laurencemoroney-blog.appspot.com/jm_blog_comments_extras.csv', extract=False)`

Navigating to the link [https://storage.googleapis.com/laurencemoroney-blog.appspot.com/jm_blog_comments_extras.csv](https://storage.googleapis.com/laurencemoroney-blog.appspot.com/jm_blog_comments_extras.csv) results in a 404 not found error.
